client/src/containers/App/App.js

```diff
@@ -56,14 +55,11 @@ class App extends Component {
           <Route path="/referrals" component={Referrals} />
           <Route path="/users" component={Users} />
           <Route path="/Volunteers" component={VolunteersForm} />
-         
           <Route path="/login" component={Login} />
           <Route
             path="/organisations-register"
             component={OrganisationsRegister}
           />
-          
-        
           <Route path="/who-we-are" component={About} />
           <Route path="/what-we-do" component={About} />
           <Route path="/who-we-support" component={About} />
```

Comment:
I think I understand what's happening - when you access the route "/shelters" both your component that has the footer gets loaded (line 54) and the main Shelters component which has all the elements you see at the bottom (line 66).

So basically, the way your components will be laid out:
<SheltersHome>
   <header>....
   <footer> ....
</SheltersHome>
<Shelters>
   main content
</Shelters>

So your footer will always be above the main content - while you want it to be the last thing before your html page ends.

I would recommend you take the footer out of the Shelthers components for now - and move it either directly in here, or under the Shelters components.